src/Components/Publishing/EditorialFeature/Components/Eoy2018Culture.tsx

```diff
@@ -1,21 +1,306 @@
+import { Box, color, Flex, Sans, Serif, space } from "@artsy/palette"
+import { unica } from "Assets/Fonts"
+import { Author, StyledAuthor } from "Components/Publishing/Byline/Author"
+import { Date, DateContainer } from "Components/Publishing/Byline/Date"
+import { Share } from "Components/Publishing/Byline/Share"
+import { getArticleFullHref } from "Components/Publishing/Constants"
+import { ArticleProps } from "Components/Publishing/Layouts/FeatureLayout"
+import { Nav, NavContainer } from "Components/Publishing/Nav/Nav"
+import { ArticleCards } from "Components/Publishing/RelatedArticles/ArticleCards/ArticleCards"
+import { CaptionContainer } from "Components/Publishing/Sections/Caption"
+import { ImageCollection } from "Components/Publishing/Sections/ImageCollection"
 import {
-  ArticleProps,
-  FeatureLayout,
-} from "Components/Publishing/Layouts/FeatureLayout"
-import { Nav } from "Components/Publishing/Nav/Nav"
+  ImageSetContainer,
+  ImageSetPreview,
+} from "Components/Publishing/Sections/ImageSetPreview"
 import React from "react"
 import styled from "styled-components"
 
-export const Eoy2018Culture: React.SFC<ArticleProps> = props => {
-  return (
-    <ArticleWrapper>
-      <Nav canFix={false} transparent />
-      <FeatureLayout {...props} />
-    </ArticleWrapper>
-  )
+interface State {
+  chapters: any[]
```

Comment:
Does this need to be a state or a private member of the class is good enough? It doesn't seem like we update the `chapters` in this component.

Reply:
Was trying to make it a static variable on the class, but got some type errors. I think maybe your suggestion of passing sections from props makes sense to avoid this.